stackslib/src/net/download/nakamoto/download_state_machine.rs

```diff
@@ -862,6 +865,14 @@ impl NakamotoDownloadStateMachine {
         self.tenure_download_schedule = schedule;
         self.tenure_block_ids = tenure_block_ids;
         self.available_tenures = available;
+
+        let highest_available_tenure = self.find_highest_available_tenure();
+        self.highest_available_tenure = highest_available_tenure;
```

Comment:
No need for the `let` binding here:

```suggestion
        self.highest_available_tenure = self.find_highest_available_tenure();
```

Reply:
Fixed